src/base58.h

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) 2009-2010 Satoshi Nakamoto
 // Copyright (c) 2009-2015 The Bitcoin Core developers
-// Copyright (c) 2014-2017 The Syscoin Core developers
+// Copyright (c) 2014-2017 The Dash Core developers
+// Copyright (c) 2014-2016 The Syscoin Core developers
```

Comment:
The copyright is longer for Dash than it is for us?   How does that work?  And according to this both copyrights have expired?